.github/workflows/joystream-types.yml

```diff
@@ -40,6 +41,9 @@ jobs:
     - name: checks
       run: |
         yarn install --frozen-lockfile --network-timeout 120000
+        yarn workspace @joystream/types build
+        yarn workspace @joystream/cd-schemas generate:all
```

Comment:
We don't need to build cd-schemas here

Reply:
indeed

Reply:
done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6